rc/tools/make.kak

```diff
@@ -60,7 +60,7 @@ define-command -hidden make-jump %{
                 set-register a "%reg{1}/%reg{2}" "%reg{3}" "%reg{4}" "%reg{5}"
             } catch %{
                 set-register / %opt{make_error_pattern}
-                execute-keys <a-h><a-l> s<ret>l
+                execute-keys n
```

Comment:
sounds reasonable. I didn't test this but I think this should be something like `<a-h><semicolon>n` to make sure `<ret>` keeps working if the cursor is in the middle of an error message.